source/aggregation-pipeline-builder.txt

```diff
@@ -31,6 +31,13 @@ aggregation pipeline. The :guilabel:`Preview of Documents in the
 Collection` section shows 20 documents randomly sampled from the chosen
 collection.
 
+.. note:: 
```

Comment:
[suggestion]

For findability, suggest giving this note a title like "Atlas Search Stages"

```suggestion
.. note:: Atlas Search Stages
```